pages/_app/index.jsx

```diff
@@ -13,6 +13,8 @@ import { initStore, GlobalProvider } from 'store'
 import Application from 'components/application'
 import { Sentry } from 'utils/sentry'
 
+import './index.css'
```

Comment:
This may create a conflict with design, files order may be different and unpredictable. I would not export this to a file here, we already had some issues files order before.

Reply:
You are right but where else should I put it. I was thinking about `public/reset-styles.css` and load it manually. Do you think it would be better than this?

Reply:
Ok let's keep it like that for now.